generated/nidaqmx/nidaqmx.proto

```diff
@@ -4431,7 +4464,7 @@ message CfgAnlgEdgeStartTrigResponse {
 message CfgAnlgMultiEdgeRefTrigRequest {
   nidevice_grpc.Session task = 1;
   string trigger_sources = 2;
-  repeated int32 trigger_slope_array = 3;
+  repeated Slope1 trigger_slope_array = 3;
```

Comment:
I think it breaks source but not binary (wire format) compatibility.

https://protobuf.dev/programming-guides/proto2/#updating

> enum is compatible with int32, uint32, int64, and uint64 in terms of wire format (note that values will be truncated if they don’t fit). However, be aware that client code may treat them differently when the message is deserialized. Notably, unrecognized enum values are discarded when the message is deserialized, which makes the field’s has.. accessor return false and its getter return the first value listed in the enum definition, or the default value if one is specified. In the case of repeated enum fields, any unrecognized values are stripped out of the list. However, an integer field will always preserve its value. Because of this, you need to be very careful when upgrading an integer to an enum in terms of receiving out of bounds enum values on the wire.

Note that this API is only supported by one device model (NI 9775) and really does not justify bumping the major version. If breaking source compatibility is unacceptable, then I think it would be preferable to add an override to https://github.com/ni/grpc-device/blob/main/source/codegen/metadata/nidaqmx/functions_addon.py to restore the old data type.

Reply:
I added the source-breaking tag to the PR. In the past we have not raised the major version for source-breaking. I am not sure if that is the right move for semantic versioning but that is the process we have been following. For binary breaking we would want to do so.